composer.json

```diff
@@ -1,108 +1,154 @@
-        "laravel/browser-kit-testing": "^1.0",
-        "laravel/framework": "5.6.*",
-        "laravel/passport": "^5.0",
-        "laravel/tinker": "~1.0",
-        "laravelcollective/html": "^5.6",
-        "league/csv": "^8.0",
-        "maatwebsite/excel": "~2.1.0",
-        "maknz/slack": "@dev",
-        "malahierba-lab/public-id": "dev-master",
-        "planetteamspeak/ts3-php-framework": "1.1.28",
-        "pusher/pusher-php-server": "~3.0",
-        "skymeyer/vatsimphp": "1.0.*",
-        "vatsim/sso": "dev-develop",
-        "vatsim/xml": "dev-master",
-        "vluzrmos/slack-api": "dev-master",
-        "watson/rememberable": "2.0.3"
+  "name": "vatsimuk/core",
+  "description": "VATSIM UK Core Framework",
+  "keywords": [
+    "vatsim",
+    "vatsimuk",
+    "vatsim-uk",
+    "vatsim uk",
+    "planes",
+    "aviation"
+  ],
+  "license": "MIT",
+  "type": "project",
+  "authors": [
+    {
+      "name": "Anthony Lawrence",
+      "email": "artisan@anthonylawrence.me.uk",
+      "homepage": "https://github.com/A-Lawrence"
+    },
+    {
+      "name": "Neil Farrington",
+      "homepage": "https://github.com/N-Farrington"
+    },
+    {
+      "name": "Joe Clifford"
     },
-    "require-dev": {
-        "fzaninotto/faker": "~1.8",
-        "mockery/mockery": "0.9.*",
-        "phpunit/phpunit": "^7.0",
-        "symfony/dom-crawler": "^4.0",
-        "symfony/css-selector": "^4.0",
-        "laravel/homestead": "^7.8.0",
-        "laracasts/generators": "^1.1",
-        "filp/whoops": "~2.0"
+    {
+      "name": "George Jones"
     },
-    "autoload": {
-        "classmap": [
-            "database/seeds",
-            "database/factories"
-        ],
-        "files": [
-            "app/helpers.php"
-        ],
-        "psr-4": {
-            "App\\": "app/"
-        }
+    {
+      "name": "Calum Towers"
     },
-    "autoload-dev": {
-        "psr-4": {
-            "Tests\\": "tests/"
-        }
+    {
+      "name": "Andy Ford"
     },
-    "extra": {
-        "laravel": {
-            "dont-discover": [
-                "laravelcollective/html"
-            ]
-        }
+    {
+      "name": "Alex Toff"
     },
-    "scripts": {
-        "post-root-package-install": [
-            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
-        ],
-        "post-create-project-cmd": [
-            "@php artisan key:generate"
-        ],
-        "post-autoload-dump": [
-            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
-            "@php artisan package:discover"
-        ],
-        "ide-helper": [
-            "php artisan ide-helper:generate",
-            "php artisan ide-helper:models -RW",
-            "php artisan ide-helper:meta"
-        ]
+    {
+      "name": "Craig Milne"
     },
-    "config": {
-        "platform": {
-            "php": "7.2"
-        },
-        "preferred-install": "dist",
-        "sort-packages": true
+    {
+      "name": "Calum Axon"
```

Comment:
You're too used to working with me! Two L's for @AxonC

Reply:
No. He shall only have one.

Reply:
🙄

Reply:
The distinction is very important! One of us can speak properly allegedly...